.github/workflows/main.yml

```diff
@@ -66,27 +64,15 @@ jobs:
       fail-fast: false
       matrix:
         ceph_version:
-        - "octopus"
-        - "pacific"
-        - "quincy"
-        - "reef"
-        - "squid"
-        - "pre-reef"
-        - "pre-squid"
-        - "main"
+        - wip-pkalever-rbd-group-snap-mirror
         go_version:
         - ${{ needs.go-versions.outputs.latest }}
-        include:
-        - ceph_version: "reef"
-          go_version: ${{ needs.go-versions.outputs.prev }}
-        - ceph_version: "reef"
-          go_version: ${{ needs.go-versions.outputs.unstable }}
     steps:
     - uses: actions/checkout@v4
     - name: Set cores to get stored as "core"
       run:  sudo bash -c 'echo "core" > /proc/sys/kernel/core_pattern'
     - name: Run tests
-      run: make test-containers-test "CEPH_VERSION=${{ matrix.ceph_version }}" "GO_VERSION=${{ matrix.go_version }}" "RESULTS_DIR=$PWD/_results"
+      run: make test-containers-test CEPH_IMG="ghcr.io/nikhil-ladha/rbd-group-mirroring" CEPH_TAG="latest" "CEPH_VERSION=${{ matrix.ceph_version }}" "GO_VERSION=${{ matrix.go_version }}" "RESULTS_DIR=$PWD/_results"
```

Comment:
instead of modifying this line, you can set the environment for the job, that feels less invasive

```yaml
env:
    CEPH_IMG: ghcr.io/nikhil-ladha/rbd-group-mirroring
    CEPH_TAG: latest
```

That should get picked up by the `make` command automatically.